controllers/idler/idler_controller.go

```diff
@@ -133,9 +138,19 @@ func (r *Reconciler) ensureIdling(ctx context.Context, idler *toolchainv1alpha1.
 	newStatusPods := make([]toolchainv1alpha1.Pod, 0, 10)
 	for _, pod := range podList.Items {
 		pod := pod // TODO We won't need it after upgrading to go 1.22: https://go.dev/blog/loopvar-preview
-		logger := log.FromContext(ctx)
-		podLogger := logger.WithValues("pod_name", pod.Name, "pod_phase", pod.Status.Phase)
+		podLogger := log.FromContext(ctx).WithValues("pod_name", pod.Name, "pod_phase", pod.Status.Phase)
 		if trackedPod := findPodByName(idler, pod.Name); trackedPod != nil {
+			// check the restart count for the trackedPod
+			crashlooping, restartCount := isRestartingOften(pod.Status)
+			if crashlooping {
```

Comment:
I got really confused by reading two returned values that look like that they represent some different number. 
let's either rename the first var to `reachedThreshold` or change it to 
```go
restartCount := getHighestRestartCount(pod.Status)
if restartCount > RestartThreshold {

Reply:
change done in [9164efe](https://github.com/codeready-toolchain/member-operator/pull/630/commits/9164efed1c4569ec9780def2f3ef716fc2da1abe)